values/vault-operator/vault-operator-raw.gotmpl

```diff
@@ -158,30 +175,42 @@ resources:
               user_claim: email
               groups_claim: groups
               # This policy will be applied to each user that uses this role
-              token_policies: [allow-list-all-secrets, allow_sandbox]
+              token_policies:
+              - allow-list-all-secrets
+              - allow_sandbox
               ttl: 1h
               #  http://localhost:8250/oidc/callback - for: vault login -method oidc
               allowed_redirect_uris: "http://localhost:8250/oidc/callback,https://vault.{{ $v.cluster.domainSuffix }}/ui/vault/auth/oidc/oidc/callback"
           - type: kubernetes
             roles:
               # Allow every pod in the vault namespace to use the secret kv store
               - name: default
-                bound_service_account_names: ["kubernetes-external-secrets", "vault", "default"]
+                bound_service_account_names: 
+                - "kubernetes-external-secrets"
```

Comment:
Nitpicking: quotes are not strictly necessary here.

Reply:
done